.github/workflows/releaseNigthly.yml

```
@@ -236,3 +236,26 @@ jobs:
       run: $GITHUB_WORKSPACE/.github/scripts/upload_failure_logs.sh
       env:
         PLATFORM_TARGET: ${{matrix.target}}
+
+
+  Trigger_Docker:
+    needs: [Linux, Macos]
+    steps:
+    - name: Checkout code
+      shell: bash
+      run: |
+        cd $HOME
+        git clone https://github.com/${REP}
+        cd ./${REP##*/}
+        git checkout --force ${GITHUB_SHA}
+        pip3 install --user --no-deps .
+      env:
+        REP: ${{github.repository}}
+    - name: Trigger docker workflow
+      shell: bash
+      run: |
+        cd $HOME
+        kiwix-build/.github/scripts/trigger_docker_workflow.py
+      env:
+        PLATFORM_TARGET: ${{matrix.target}}
+        GITHUB_PAT: ${{secrets.GHCR_TOKEN}}
```

I created the corresponding secret (name cannot start with `GITHUB_`…)
```suggestion
        GITHUB_PAT: ${{secrets.DOCKER_TRIGGER_GITHUB_PAT}}
```

---

`GHCR_TOKEN` is the secret containing the token to trigger the docker workflows:
- it can be removed from linux job now (the whole `GITHUB_PAT` env line)
- I moved it to `DOCKER_TRIGGER_GITHUB_PAT` so its usage is more clear.
- I will then remove the repo-level `GHCR_TOKEN` from `kiwix-build` secrets. There's a `kiwix`-wide `GHCR_TOKEN` that is different (we were overwriting it) and is used to push packages to ghcr.io